### DISCUSSION TITLE
2.0.1 release plan?

### DISCUSSION THREAD

**Discussion by junnplus** [apache/airflow#13909] [Q&A] [answered]
rt, Is there a release plan for 2.0.1? when ?

**Comment by turbaszek** [comment, marked as answer]
Please see the notes from last DevCall:
https://lists.apache.org/thread.html/r9542d934eea96c7c216ec14b55d7c74612720750cbcf2f0d24ad725a%40%3Cdev.airflow.apache.org%3E

You may also observe the milestone:
https://github.com/apache/airflow/milestone/23

As you may see the date is planed for 8th of February 😉
